shared/css/color.scss

```diff
@@ -59,6 +62,7 @@ $hdr_color:   $purple;
 // Some extra colors.
 $red: #c00;
 $lightest_red: #fcc;
+$redesign_red: #d5001a;
```

Comment:
Would be interested to get @tanyaparker and @breville 's takes on these colors (both if we need new ones, and if these are the right names for them).